### ISSUE TITLE
Codex is broken since cabal-2.2.0.0

### ISSUE THREAD

**Issue by hasufell** [aloiscochard/codex#89]
```
$ codex update
codex: Run the 'configure' command first.
codex: *warning* falling back on dependency resolution using hackage
codex: /home/maerwald/.cabal/packages/hackage.haskell.org/00-index.tar: openBinaryFile: does not exist (No such file or directory)
```

`00-index.tar` doesn't exist anymore, only `01-index.tar` is created.

**Comment by hasufell** [comment]
https://github.com/aloiscochard/codex/pull/88

**Comment by LucianU** [comment]
What's the status on this?

**Comment by parsonsmatt** [comment]
The most recent release should have fixed this.

**Comment by seanparsons** [comment]
@parsonsmatt The most recent release of codex? I just tried it with master and it still looks to be broken to me.

**Comment by parsonsmatt** [comment]
Indeed. This is what I get:

```
λ ~/Projects/rowdy/rowdy/ master codex --version
codex: 0.5.2.0

λ ~/Projects/rowdy/rowdy/ master codex update
codex: Run the 'configure' command first.
codex: *warning* falling back on dependency resolution using hackage
codex: /home/matt/.cabal/packages/hackage.haskell.org/01-index.tar/01-index.tar: openBinaryFile: inappropriate type (Not a directory)
```

**Comment by hasufell** [comment]
This is still broken after a year.

**Comment by parsonsmatt** [comment]
Should be fixed by #102 .
